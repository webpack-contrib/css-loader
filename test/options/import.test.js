/* eslint-disable
  prefer-destructuring,
*/
import webpack from '../helpers/compiler';

describe('Options', () => {
  describe('import', () => {
    test('{Boolean}', async () => {
      const config = {
        loader: {
          test: /\.css$/,
          options: {},
        },
      };

      const stats = await webpack('imports/fixture.js', config);
      const { source } = stats.toJson().modules[1];

      expect(source).toMatchSnapshot();
    });
  });
});
