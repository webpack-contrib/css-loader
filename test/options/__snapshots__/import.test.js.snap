// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options import {Boolean} 1`] = `
"// CSS Imports
import CSS__IMPORT__0 from './import.css';


// CSS Exports
// CSS Modules Exports


// CSS
export default \`.fixture {
  width: 100%;
}
\`"
`;
